src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -121,7 +121,7 @@
        {:key (str "action-" (or image icon))}))])
 
 (defn toolbar
-  ([props nav-item content-item] (toolbar props nav-item content-item [actions [{:image :blank}]]))
+  ([props nav-item content-item] (toolbar props nav-item content-item nil))
```

Comment:
hm

Reply:
That `blank` was needed before to keep proper balance when no actions. Maybe it's not necessary anymore? #cleanup :D

Reply:
yeah i think we should rework toolbar component, i don't like these nils tbh

Reply:
agree!